### ISSUE TITLE
KeyError: 'computes'

### ISSUE THREAD

**Issue by noplay** [GNS3/gns3-server#1059]
https://sentry.io/gns3/gns3-server/issues/287216499/

```
KeyError: 'computes'
(7 additional frame(s) were not displayed)
...
  File "aiohttp/signals.py", line 48, in send
    yield from self._send(*args, **kwargs)
  File "aiohttp/signals.py", line 16, in _send
    yield from res
  File "gns3server/web/web_server.py", line 195, in _on_startup
    yield from Controller.instance().start()
  File "gns3server/controller/__init__.py", line 73, in start
    computes = yield from self._load_controller_settings()
  File "gns3server/controller/__init__.py", line 179, in _load_controller_settings
    return data["computes"]

KeyError: 'computes'
```

**Comment by noplay** [comment]
I think If computes doesn't exist we need to default to a value. Not sure if it a dict or a list

**Comment by ziajka** [comment]
@noplay it seems to be a list:
```

{
    "auto_close": true,
    "auto_open": false,
    "auto_start": false,
    "name": "HelloWorld",
    "project_id": "68f1a08c-3f57-480f-a0b0-fa4f44055da1",
    "revision": 8,
    "scene_height": 1000,
    "scene_width": 2000,
    "topology": {
        "computes": [],
        "drawings": [],
        "links": [],
        "nodes": []
    },
    "type": "topology",
    "version": "2.0.2"
}
```

**Comment by noplay** [comment]
You can make a Pull Request for it

**Comment by ziajka** [comment]
Fix #1062

**Comment by ziajka** [comment]
Fix #1062
